### DISCUSSION TITLE
两个建议

### DISCUSSION THREAD

**Discussion by Geeyun-JY3** [the1812/Bilibili-Evolved#1346] [功能建议 / Ideas]
如果开启了自定义顶栏，动态详情页的顶栏的这个发布动态就看不到了。做个 FAB？
![image](https://user-images.githubusercontent.com/49396421/102794590-91a54a80-43e6-11eb-84fc-5cbff8f2265d.png)
暂时关闭 `自动从历史记录点播放` 功能后这个`允许跨集跳转` 设置项应该也隐藏吧（还有个缩进，看上去以为在 `自动展开视频简介` 下）
![image](https://user-images.githubusercontent.com/49396421/102794095-cfee3a00-43e5-11eb-9473-7a2b51e9bad5.png)

**Comment by the1812** [comment]
> 这个发布动态就看不到了

我觉得真想发动态直接去动态首页发就好了, 顶栏没必要有这个

> 这个`允许跨集跳转`设置项应该也隐藏吧

忘了隐藏了...

**Reply by Geeyun-JY3** [reply]
> 我觉得真想发动态直接去动态首页发就好了, 顶栏没必要有这个

我的意思倒不是在顶栏加。
是在动态详情页原版顶栏有这么个按钮，如果要做个对应的功能，我建议做个 FAB，仅在自定义顶栏开启时出现。如果你觉得没必要，那就不加吧，我无所谓。

**Reply by the1812** [reply]
所以FAB是啥。。

**Reply by Geeyun-JY3** [reply]
> 所以FAB是啥。。

啊你用 Material Design 不知道 FAB😂？就一个悬浮的加号啊：https://www.mdui.org/design/components/buttons-floating-action-button.html#buttons-floating-action-button-floating-action-button

**Reply by the1812** [reply]
我只是嫖一下图标, 当然不了解里面那些组件(

**Comment by Geeyun-JY3** [comment]
1. 最终动态详情页也没改成图中样子
2. 这 bug 当时应该修了，不过该功能现在也已经废弃了：https://github.com/the1812/Bilibili-Evolved/releases/tag/v1.11.5
https://github.com/the1812/Bilibili-Evolved/blob/master/registry/lib/deprecated/auto-continue/index.ts
